documentation/1.2/tour/interop.md

```diff
@@ -85,7 +85,107 @@ Aether. You can find more information [here](../../reference/repository/maven).
 
 ## Deploying Ceylon on OSGi
 
-_TODO: document OSGi interop._
+Ceylon is fully interoperable with OSGI, so that Ceylon modules:
```

Comment:
This list isn’t a list on the website, probably needs one blank line before the first `-` list item line.